tests/test_objective_funs.py

```diff
@@ -169,6 +170,21 @@ def test(eq):
         test(Equilibrium(iota=PowerSeriesProfile(0)))
         test(Equilibrium(current=PowerSeriesProfile(0)))
 
+    @pytest.mark.unit
+    def test_target_shear(self):
+        """Test calculation of shear profile."""
+
+        def test(eq):
+            obj = Shear(target=1, weight=2, eq=eq)
+            obj.build()
+            shear = obj.compute_unscaled(*obj.xs(eq))
+            shear_scaled = obj.compute_scaled_error(*obj.xs(eq))
+            np.testing.assert_allclose(shear, 0)
+            np.testing.assert_allclose(shear_scaled, -2 / np.sqrt(3))
+
+        test(Equilibrium(iota=PowerSeriesProfile(0)))
```

Comment:
might want to test with a nonzero iota to make sure the non-degenerate case is correct as well